frontend/src/components/SortableTaskList.tsx

```diff
@@ -28,6 +36,45 @@ export const SortableTaskList: FC<{
 }) => {
   const { t } = useTranslation();
   const [search, setSearch] = useState('');
+  const [searched, setSearched] = useState(tasks);
+  const listRef = useRef<VariableSizeList<any> | null>(null);
+  const [divRef, setDivRef] = useState<HTMLDivElement | null>(null);
+  const [rowHeights, setRowHeights] = useState<number[]>([]);
+
+  useEffect(() => {
+    setSearched(
+      tasks.filter(({ name }) => name.toLowerCase().includes(search)),
+    );
+  }, [search, tasks]);
+
+  useEffect(() => {
+    if (!divRef) return;
+    const heights = searched.map(({ name }) => {
+      divRef.textContent = name;
+      const textHeight = divRef.offsetHeight;
+      divRef.textContent = '';
+
+      return 22 + textHeight; // 22 = margin + padding
+    });
+    setRowHeights(heights);
+    listRef.current!.resetAfterIndex(0);
+  }, [searched, divRef]);
+
+  const Row = ({ data, index, style }: RowProps) => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { height: _, ...rest } = style;
+    return (
+      <SortableTask
+        style={{ ...rest }}
+        key={data[index].id}
+        task={data[index]}
+        index={index}
+        disableDragging={disableDragging}
+        showRatings={!!showRatings}
+      />
+    );
+  };
```

Comment:
This is to avoid weird error msg: https://github.com/atlassian/react-beautiful-dnd/issues/1647#issuecomment-686956052